README.md

```diff
@@ -135,19 +111,17 @@ Netlify, Github Pages
 
 👤 **Daniel Selase Anyidoho**
 
-- GitHub: [@githubhandle](https://github.com/dselasea)
-- Twitter: [@twitterhandle](https://twitter.com/dselasea)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/dselasea)
+- GitHub: [@dselasea](https://github.com/dselasea)
+- Twitter: [@dselasea](https://twitter.com/dselasea)
+- LinkedIn: [Daniel Selase Anyidoho](https://linkedin.com/in/dselasea)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [Filter Scores] **Filtering Scores**
-- [Edit and Update] **Edit Name and Score**
-- [Date and Time] **Date and Time**
+**Filtering Scores**
+**Edit Name and Score**
+**Date and Time**
```

Comment:
- [ ] [optional] @dselasea it would be nice if your key features and future features were listed as a list, instead of them being in the same letter, it will make then look cleaner.